src/styles/block/block.scss

```diff
@@ -0,0 +1,28 @@
+.block {
+  display: grid;
+  grid-template-rows: repeat(2, minmax($box-size, 1fr));
```

Comment:
I think you don't really need to set rows here
```suggestion
```

Reply:
> I think you don't really need to set rows here

Right. Have removed them and nothing has changed:)